homeassistant/components/nextcloud/sensor.py

```diff
@@ -76,6 +587,14 @@ class NextcloudSensor(NextcloudEntity, SensorEntity):
     """Represents a Nextcloud sensor."""
 
     @property
-    def native_value(self) -> StateType:
+    def native_value(self) -> StateType | datetime:
         """Return the state for this sensor."""
-        return self.coordinator.data.get(self.item)
+        val = self.coordinator.data.get(self.item)
+        if self.item == "system cpuload":
```

Comment:
I can't see any item in the list that has this value.

Reply:
81e58a5